src/dashboard/trulens/dashboard/pages/Evaluations.py

```diff
@@ -389,12 +403,45 @@ def highlight(s):
                             df.meta.apply(lambda m: pd.Series(m))
                         ).drop(columns="meta")
 
-                        st.dataframe(
-                            df.style.apply(highlight, axis=1).format(
-                                "{:.2f}", subset=["result"]
+                        if "groundedness" in feedback_name.lower():
```

Comment:
This will not work in the case where user names the groundedness feedback function to str not containing `"groundedness"` in it right?

Reply:
That's true. Could also cause an issue if a non-groundedness feedback function for some reason was named groundedness.

I think I'm okay with this, but open to suggestions.

Reply:
Can we add a note / comment in the code and we can come back to this or mark as a TODO for the community?